src/app/components/game/interface/menu-bar/settings/volume/volume-slider.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { GameSettingsService, StorageKey } from 'src/app/services/game-settings.service';
+
+@Component({
+  selector: 'app-menu-settings-volume-slider',
+  templateUrl: './volume-slider.component.html',
+  styleUrls: ['./volume-slider.component.css'],
+})
+export class VolumeSlider implements OnInit, OnDestroy {
+  public MAX_VOLUME = 1;
+  public MIN_VOLUME = 0;
+  public VOLUME_STEPS = 0.05;
+
+  volumeMusic: number;
+  volumeSoundEffects: number;
+
+  private subscriptionVolume: Subscription;
+  private subscriptionSoundEffects: Subscription;
+
+  constructor(public gss: GameSettingsService) {}
+
+  ngOnInit(): void {
+    this.subscriptionVolume = this.gss.musicVolume.subscribe((volume: number) => {
+      this.volumeMusic = volume;
+    });
+    this.subscriptionSoundEffects = this.subscriptionVolume = this.gss.soundEffectVolume.subscribe((volume: number) => {
```

Comment:
ist das richtig? Ich glaube du überschreibst hier this.subscriptionVolume mit der Subscription für die Effekte.